### PR TITLE
{vis}[intel/2017a] Pango v1.40.10, HarfBuzz v1.4.8

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#5037~~, #5038, ~~#5040~~, ~~#5041~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d0bf9f0ee59cdef8116e973e77150fc4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6d291c2b12bf6c077d0602f678409234 for a full test report.

**Comment by boegel** [comment]
@easybuilders/easybuild-easyconfigs-maintainers please review (dep for #5045)

File: easybuild/easyconfigs/p/Pango/Pango-1.40.10-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ('cairo', '1.14.10'),
+    ('HarfBuzz', '1.4.8'),
+]
+
+preconfigopts = "./autogen.sh && "
+configopts = "--disable-silent-rules --enable-introspection=yes --enable-static --enable-shared "
+
+modextrapaths = {
+    'XDG_DATA_DIRS': 'share',
+}
```
Comment (verdurin) [inline easybuild/easyconfigs/p/Pango/Pango-1.40.10-intel-2017a.eb]: @boegel Please add a sanity check.

**Comment by boegel** [comment]
No longer relevant due to old toolchain

**Pull request closed without merging**